cdc/sink/dispatcher/expression.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2022 PingCAP, Inc.
```

Comment:
Better change file name to topic_expression? 

We would prefer to have files that distinguish between partition and topic. Maybe it would be better to create two folders to separate them.